internal/contour/translator_test.go

```diff
@@ -344,7 +347,7 @@ func TestTranslatorAddIngress(t *testing.T) {
 			Domains: []string{"*"},
 			Routes: []*v2.Route{{
 				Match:  prefixmatch("/"),
-				Action: clusteraction("default/backend/80"),
+				Action: clusteraction("default/backend/80", nil),
```

Comment:
rather than use nil everywhere, please make a new helper, clusteractiontimeout("cluster", timeout), the don't use it here and everywhere so we get the default :)